cmd/snap/cmd_help.go

```diff
@@ -198,7 +198,7 @@ var helpCategories = []helpCategory{
 	}, {
 		Label:       i18n.G("Permissions"),
 		Description: i18n.G("manage permissions"),
-		Commands:    []string{"interfaces", "interface", "connect", "disconnect"},
+		Commands:    []string{"connections", "interfaces", "interface", "connect", "disconnect"},
```

Comment:
should we stop listing "interfaces" here?  wait for 2.39 ?  cc @chipaca

Reply:
I think we should drop it (it's confusing, and this is better). Further, I think `snap interfaces` should print a 'try the new connections' to stderr.